src/main.rs

```diff
@@ -50,6 +50,14 @@ fn parse_matches() -> clap::ArgMatches {
                 .args(&["local_path", "release_channel"])
                 .required(true),
         )
+        .arg(
+            Arg::with_name("validator_lab_directory")
+                .long("validator-lab-dir")
+                .takes_value(true)
+                .required(true)
+                .help("Absolute path to validator lab directory. 
+                e.g. /home/sol/validator-lab"),
+        )
```

Comment:
I think we're solving the issue of using this tool by an executable file. If we have this one, it also means we will always need to clone validator-labs. If we have a path `cargo install` for this tool in the future, it will be a little bit confused haha.

I'm thinking maybe just like the Dockerfile, we get those files in line 🙈

Reply:
> I'm thinking maybe just like the Dockerfile, we get those files in line

oooh can you elaborate a little more please? Are you saying we should add those startup scripts inline? Like in the rust code much like the dockerfile is created inline?

Reply:
See this PR: https://github.com/anza-xyz/validator-lab/pull/9. I hardcoded the scripts. Think what you're saying makes sense slash is the right approach. See: https://github.com/anza-xyz/validator-lab/pull/9/files#diff-42d26d9f54d52fc46e96e03003a53843bea5e35c56ce03ade3ee30d0654b2320